app/ide-desktop/lib/client/src/config/parser.ts

```
@@ -0,0 +1,319 @@
+    const rightWidth = totalWidth - leftWidth
+
+    for (const [groupName, section] of Object.entries(sections)) {
+        if (section.entries.length > 0) {
+            console.log('\n\n')
+            const groupTitle = chalk.bold(`${naming.camelCaseToTitle(groupName)} Options `)
+            console.log(groupTitle)
+            const description = wordWrap(section.description, totalWidth).join('\n')
+            console.log(description)
+            console.log()
+            section.entries.sort()
+            for (const [cmdOption, option] of section.entries) {
+                const indent = ' '.repeat(indentSize)
+                let left = indent + chalk.bold(chalk.green(optionPrefix + cmdOption))
+                const spaces = ' '.repeat(leftWidth - stringLength(left))
+                left = left + spaces
+
+                let firstSentenceSplit = option.description.indexOf('. ')
+                let firstSentence =
+                    firstSentenceSplit == -1
+                        ? option.description
+                        : option.description.slice(0, firstSentenceSplit + 1)
+                let otherSentences = option.description.slice(firstSentence.length)
+
+                const def = option.defaultDescription ?? option.default
+                let defaults = ''
+                if (def != null && def !== '') {
+                    defaults = ` Defaults to ${chalk.green(def)}.`
+                }
+                let description = firstSentence + defaults + chalk.gray(otherSentences)
+                const lines = wordWrap(description, rightWidth).map(
+                    line => line + ' '.repeat(rightWidth - stringLength(line))
+                )
+                const right = lines.join('\n' + ' '.repeat(leftWidth))
+                console.log(left + right)
+            }
+        }
+    }
+}
+
+function wordWrap(str: string, width: number): string[] {
+    if (width <= 0) {
```

Shouldn't there at least be a warning? Silently discarding input might be difficult to debug later.